src/V3AstNodes.h

```diff
@@ -6092,6 +6095,51 @@ class AstFEof final : public AstNodeUniop {
     AstNode* filep() const { return lhsp(); }
 };
 
+class AstStdRandomize final : public AstNodeMath {
+    // Randomize the specified variable (std::randomize)
+    // op1p is reference to var being randomized
+    // op2p is reference to table containing enum values (if randomized var is enum)
+private:
+    typedef std::unordered_map<AstNodeDType*, AstVar*> ValueTableMap;
+    static ValueTableMap m_enumValueTabMap;  // Tables with enum values
+    static AstVar* enumValueTabp(AstEnumDType* nodep);
+
+    // Reference to enum type (if randomized var is enum), needed here because at emit stage type
+    // info is missing if enum has specified base type
+    AstEnumDType* m_enumDTypep = nullptr;
+    AstMemberDType* m_varMemberp = nullptr;  // Var member to randomize (if var is of complex type)
+    int m_varBitOffset = 0;  // Bit offset of var; used for structs contained within structs
+    void enumVarPrep(FileLine* fl);
```

Comment:
We should be able to get rid of most/all of this as will be local to the randomize visitors.  Generally pointers to other nodes should be avoided unless no other choice as they need patching up at many later stages so are a maintenance problem.

Reply:
I put it here for future use with `std::randomize`, but as I said in another comment I'll probably remove AstStdRandomize completely at this point.